src/sentry/static/sentry/app/components/events/interfaces/threads/getRelevantFrame.ts

```diff
@@ -0,0 +1,18 @@
+import {Frame} from 'app/types/events';
```

Comment:
I think we are using .tsx extension everywhere, even if the file does not contain jsx.